### ISSUE TITLE
[Desktop] Hero Section

### ISSUE THREAD

**Issue by xReady45** [UnlockdFinance/unlockd-landing-page#30]
Video https://drive.google.com/file/d/1EojpuScBgUh7itoKGBnYj6wY8M6kYdfr/view?usp=sharing

**Comment by shaheerhashim10** [comment]
- [ ] Adjust Header gradient
- [ ] Adjust the text of the Hero section
- [ ] Adjust Typography of Header
- [ ] Add a Placeholder from Figma in the Header

**Comment by xReady45** [comment]
Yeah, I'm waiting until we have everything fixed to see how we fit the placeholder that's currently missing

**Comment by shaheerhashim10** [comment]
@xReady45 all of the above fixes are done. Please review them and give feedback.
